pkg/operator/ceph/pool/controller.go

```diff
@@ -186,7 +187,7 @@ func createPool(context *clusterd.Context, p *cephv1beta1.Pool) error {
 
 	// create the pool
 	logger.Infof("creating pool %s in namespace %s", p.Name, p.Namespace)
-	if err := ceph.CreatePoolWithProfile(context, p.Namespace, *p.Spec.ToModel(p.Name), p.Name); err != nil {
+	if err := ceph.CreatePoolWithProfile(context, p.Namespace, *p.Spec.ToModel(p.Name), appName); err != nil {
```

Comment:
Have you verified that `createPool()` is only called when a "single" Pool CRD object is created?

Reply:
`createPool()` is only called by `PoolController.onAdd()` and `PoolController.onUpdate()`, so it's only relevant to `Pool` k8s objects.